lib/src/config/builder.rs

```diff
@@ -89,7 +89,9 @@ impl ConfigBuilder {
     /// assert_eq!(config.address.as_str(), "127.0.0.1");
     /// ```
     pub fn address<A: Into<String>>(mut self, address: A) -> Self {
-        self.address = address.into();
+        let address = address.into();
+        // FIXME: Remove unwrap
+        self.address = address.parse().unwrap();
```

Comment:
I'd like to get rid of this `unwrap()` but not sure what's the best way to do it.

Reply:
Keep the address as a `String` in the struct. Errors aren't handled in the individual builders. They're handled at the very end by `finalize()`.